model/Core/Classes/Hash.md

```diff
@@ -8,12 +8,11 @@ A mathematically calculated representation of a grouping of data.
 
 ## Description
 
-A hash is a grouping of characteristics unique to the result
-of applying a mathematical algorithm
-that maps data of arbitrary size to a bit string (the hash)
-and is a one-way function, that is,
-a function which is practically infeasible to invert.
-This is commonly used for integrity checking of data.
+A hash is a grouping of characteristics unique to the result of applying a mathematical algorithm
+that maps data of arbitrary size to a bit string (the hash) and is a one-way function, that is,
+a function which is practically infeasible to invert. This is commonly used for integrity checking of data.
+
+The recommended method to verify the integrity of `SoftwareArtifacts` Elements (including `Files`, `Snippets`, and `Packages`) is to use the SoftwareArtifact’s `contentIdentifier` property. 
```

Comment:
I do not think that these sort of case specific details should be placed here.
This is the general structure to be used across SPDX.
Placing case specific variations here is likely to lead to an eventual laundry list of such specializations.

I propose that it would be more appropriate to add this note to the markdown files for the case specific structures (SoftwareArtifact, File, Package, Snippet).